approach/tooling.md

```diff
@@ -106,6 +106,11 @@ Using Edit \> Trim, it can be hard to control how fast the yellow Trim brackets
 -   communications and technical writing (RMarkdown, bookdown, blogdown, Quarto)
 -   [RStudio and GitHub setup instructions](https://openscapes.github.io/series/how-to/setup-rstudio-github.html)
 
+## Python
+
+- [earthaccess](https://github.com/nsidc/earthaccess): Join us for our
+  [bi-weekly hack days](https://earthaccess.readthedocs.io/en/latest/work-with-us/#bi-weekly-alternating-weeks-earthaccess-hack-days)!
```

Comment:
Link is broken: https://earthaccess.readthedocs.io/en/latest/contributing/our-meet-ups/